examples/er-rest-example/project-conf.h

```diff
@@ -93,6 +96,13 @@
 #undef COAP_PROXY_OPTION_PROCESSING
 #define COAP_PROXY_OPTION_PROCESSING   0
 
+/* Turn of DAO ACK to make code smaller */
+#undef RPL_CONF_WITH_DAO_ACK
+#define RPL_CONF_WITH_DAO_ACK          0
```

Comment:
Ok. Not a good thing to do - these examples rely on a button press and wismote seems not to react to button press. So yes I will keep it at no-ack for now.